app/helpers/appropriate_body_helper.rb

```diff
@@ -41,13 +41,35 @@ def summary_card_for_teacher(teacher)
           {
             key: { text: "Status" },
             # FIXME: this is a placeholder as we cannot display a real status yet
-            value: { text: govuk_tag(text: "placeholder", colour: %w[grey green red purple orange yellow].sample) },
+            value: { text: govuk_tag(**induction_status_tag_kwargs(teacher)) },
           },
         ]
       )
     end
   end
 
+private
+
+  def induction_status_tag_kwargs(teacher)
+    case teacher.trs_induction_status
+    when 'RequiredToComplete'
+      { text: 'Required to complete', colour: 'grey' }
```

Comment:
```suggestion
    when 'RequiredToComplete'
      { text: 'Required to complete', colour: 'yellow' }
```

Maybe yellow or dark blue, following the screenshot below? Again, feels quite different to the other grey tags? :)